### PR TITLE
Feature/AT-6325 Vue E2E testing framework - nightwatch

### PR DESCRIPTION
**Pull request by jGonzalez-CCPO**
Vue E2E testing framework - Evaluate option https://nightwatchjs.org/

Ticket: AT-6325

### CODE REVIEW

**Comment by jGonzalez-CCPO** [comment]
e2e github action required credentials to be set on the environment.

**Review by jasonburkert-ccpo — CHANGES_REQUESTED** [review]
The file `browserstack_config/nightwatch.conf.js` is not being linted during `npm run lint`.  It is however being linted during GitHub check.

I suggest:
- change local linting to be more inclusive.  this file shouldn't be missed in its current location.
- perhaps move file `nightwatch.conf.js` to `tests`

File: tests/e2e/browserstack_config/nightwatch.conf.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
+const path = require("path");
+const deepmerge = require("deepmerge");
+const chromedriver = require("chromedriver");
```
Comment (kylelaker-ccpo) [inline tests/e2e/browserstack_config/nightwatch.conf.js:3]:
```suggestion
/* eslint-disable no-no-var-requires */
const path = require("path");
const deepmerge = require("deepmerge");
const chromedriver = require("chromedriver");
/* eslint-enable no-no-var-requires ​*/
```

Let's try this

File: tests/e2e/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (jGonzalez-CCPO) [inline tests/e2e/.eslintrc.js:1]: @kylelaker-ccpo @jasonburkert-ccpo this file was generated by the vie-cli when I add Nightwatch maybe the problems is this is been ignore by github/super-linter@v4

**Comment by kylelaker-ccpo** [comment]
@palaparthi-ccpo has taken over the work on this branch for the most part; I am going to close this particular PR so that a new one can be opened.

**Pull request closed without merging**